pkgs/development/libraries/glibc/default.nix

```diff
@@ -36,6 +36,7 @@ in
       if [ -f ${stdenv.cc.cc}/lib/libgcc_s.so.1 ]; then
           mkdir -p $out/lib
           cp ${stdenv.cc.cc}/lib/libgcc_s.so.1 $out/lib/libgcc_s.so.1
+          patchelf --set-rpath $out/lib --force-rpath $out/lib/libgcc_s.so.1
```

Comment:
Yeah I copied the force-rpath from the make-bootstrap-tools figuring it was
needed. I can try without it.

On Sun, May 31, 2015, 01:57 Vladimír Čunát <notifications@github.com> wrote:

> In pkgs/development/libraries/glibc/default.nix
> <https://github.com/NixOS/nixpkgs/pull/8082#discussion_r31388350>:
>
> > @@ -36,6 +36,7 @@ in
> >        if [ -f ${stdenv.cc.cc}/lib/libgcc_s.so.1 ]; then
> >            mkdir -p $out/lib
> >            cp ${stdenv.cc.cc}/lib/libgcc_s.so.1 $out/lib/libgcc_s.so.1
> > +          patchelf --set-rpath $out/lib --force-rpath $out/lib/libgcc_s.so.1
>
> I don't see why --force-rpath is used. (I see now there's a nuked ref in
> RPATH unless we patchelf the lib.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/8082/files#r31388350>.
>